pkg/probo/document_service.go

```diff
@@ -435,12 +447,26 @@ func (s *DocumentService) SendSigningNotifications(
 					}.Encode(),
 				}
 
+				tmpl, err := template.New("signRequest").Parse(signEmailTemplate)
+				if err != nil {
+						panic(err)
+				}
+
+				data := SignEmailData{
+						SignURL: signRequestURL.String(),
+				}
+
+				var body bytes.Buffer
+				if err := tmpl.Execute(&body, data); err != nil {
+						panic(err)
```

Comment:
We should not use panic here, instead we have to return a wrapped error. In that layer we never use any panic in runtime code.